src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml

```diff
@@ -58,9 +58,9 @@ sylius:
         edit_product: Edit product
         edit_product_attribute: Edit product attribute
         edit_product_option: Edit product option
+        edit_product_variant: Edit product variant
         edit_promotion: Edit promotion
         edit_promotion_coupon: Edit promotion coupon
-        edit_coupon: Edit coupon
```

Comment:
I've removed this translation, because it was duplicated [here](https://github.com/Sylius/Sylius/pull/5126/files#diff-700ac7d934d5990dad53037acb46869cR54)